flexget/plugins/input/npo_watchlist.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import urllib
+
+from jinja2 import TemplateSyntaxError
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.task import Task
+from flexget.utils.search import normalize_unicode
+
+from bs4 import BeautifulSoup
+import unicodedata
+
+import requests
+
+import re
+from datetime import datetime, timedelta
+
+log = logging.getLogger('search_npo')
+search_results_regex = re.compile('<a href="([^"]+)" data-scorecard="{&quot;name&quot;:&quot;zoeken')
+
+class NPOWatchlist(object):
+    """
+        Produces entries for every episode on the user's npo.nl watchlist (Dutch public television).
+        Entries can be downloaded using https://bitbucket.org/Carpetsmoker/download-npo
+        
+        If 'remove_accepted' is set to 'yes', the plugin will delete accepted entries from the watchlist after download is complete.
+        
+        For example:
+            npo_watchlist:
+              email: aaaa@bbb.nl
+              password: xxx
+              remove_accepted: yes
+            accept_all: yes
+            exec:
+              fail_entries: yes
+              auto_escape: yes
+              on_output: 
+                for_accepted:
+                  - download-npo -o "path/to/directory/{{series_name_plain}}" -f "{serie_titel} - {datum} {aflevering_titel} ({episode_id})" -t {{url}}
+        """
+    
+    schema = {
+        'type': 'object',
+        'properties': {
+            'email': {'type': 'string'},
+            'password': {'type': 'string'},
+            'remove_accepted': {'type': 'boolean', 'default': False}
+        },
+        'required': [],
```

Comment:
Are email and password supposed to be in here? This being empty is why the tests are failing.